src/About/about.css

```diff
@@ -0,0 +1,90 @@
+body{
+  background-image: url(./images/About\ Me.jpg);
+  background-repeat: no-repeat;
+  background-size: 1500px 800px;
+  color: #555f77;
+  background-attachment: fixed;
+  font-family: 'Roboto', sans-serif;
+}
+
+a{
+  border-bottom: 4px solid #e4dfdf;
+  text-decoration: none;
+  color: #555f77;
+  transition: border 0.3s;
+  padding-bottom: 22px;
+}
+
+a:hover {
+  border-bottom: 4px solid;
+}
+
+#header-container {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  height: 46px;
+  z-index: 1050;
+  background-color: #e4dfdf;
+  box-shadow: 0px 2px 10px 0px black;
+  padding: 10px 20px;
+  overflow: hidden;
+}
+
+.header-fix {
+  height: 65px;
+}
+
```

Comment:
your content divs are a bit stiff. they don't move with the page size change. you can change that by adding flex to their parent element `.main-container`.  try:
`.main-container {`
`display: flex;`
    `flex-direction: column;`
    `justify-content: center;`
  `  align-items: center`
`}`